less/select.less

```diff
@@ -49,7 +49,7 @@
 
 // clear "x" button
 @select-clear-size:                 floor((@select-input-height / 2));
-@select-clear-color:                #999;
```

Comment:
Please make it only for the unique input, as this affects to all inputs

| Before | After |
| --- | --- |
| ![screen shot 2018-11-26 at 10 41 22 am](https://user-images.githubusercontent.com/1847316/49028437-07874b00-f168-11e8-97df-d599bf6ff00e.png) |  ![screen shot 2018-11-26 at 10 41 33 am](https://user-images.githubusercontent.com/1847316/49028435-07874b00-f168-11e8-8865-0763edb5d121.png) |

Reply:
Yeah, I talked to UX team about that and they are expecting to do that change on all of the selects